dcos.yml

```diff
@@ -17,13 +17,15 @@
 
 - name: DC/OS Requirements
   hosts: all
+  environment: "{{proxy_env}}"
```

Comment:
why using environment variable if there is group_vars?

Reply:
oh just I see you want to use proxy env vars for the ansible tasks? But still `proxy_env` is setting overall environment variables not just the proxy ones. So the name is confusing